### ISSUE TITLE
Enhancement Request - Add trusted mac addresses in ndsctl json command

### ISSUE THREAD

**Issue by ClydeAmazing** [nodogsplash/nodogsplash#527]
Hello Team NDS,

I wonder if we could make `ndsctl json` return list of trusted MAC addresses. Currently ndsctl json (if no client mac, ip, token is passed as a parameter) only returns authenticated and preauthenticated client details.
I have a python script that dynamically trust/untrust specific MAC addresses based on some external qualifications, it would be very helpful to easily retrieve trusted MAC addresses in JSON format.

Thank you so much.

**Comment by bluewavenet** [comment]
@ClydeAmazing 
You can get this information using `ndsctl status`, but having it in a better format would perhaps be useful.
As NDS trusted devices are neither preauthenticated nor authenticated, perhaps nds json is not the correct place for this. Perhaps a new `ndsctl trusted` command?
As this will be just a simple list, does it need to be in json format?

**Comment by ClydeAmazing** [comment]
@bluewavenet 

Well, I think it doesn't have to be in JSON format. A simple list-like structure will do, as long as it is easily parsable. 

 Creating another command could also be a good idea, although for me its kinda easier to access all NDS information including trusted MAC Addresses on a single `ndsctl json` command since the output can be easily parsed.

But any solutions that would be both practical and easier to implement on your end would do.

Thank you so much.

**Comment by bluewavenet** [comment]
@ClydeAmazing 
What about returning a json array? Something like:
```
{
"trusted:[
	"00:12:aa:bb:ef:ea",
	"00:12:aa:bb:ef:eb"
	]
}

```

**Comment by bluewavenet** [comment]
@ClydeAmazing 
Implemented in PR #529
